content/en/security/_index.md

```diff
@@ -47,31 +50,33 @@ Bring speed and scale to your production security operations. Datadog Security d
 
 Datadog Security includes [Application Security Management](#application-security-management), [Cloud SIEM](#cloud-siem), [Cloud Security Posture Management (CSPM)](#cloud-security-posture-management), and [Cloud Workload Security (CWS)](#cloud-workload-security).
 
+Check out the [30-second Product Guided Tour](https://www.datadoghq.com/guided-tour/security/) to see more.
```

Comment:
Adding this to the Further Reading and the Overview might be enough push for the user. I recommend removing it from the sub sections.
```suggestion
In Datadog, see the [30-second Product Guided Tour](https://www.datadoghq.com/guided-tour/security/) to learn  more.
```

Reply:
Thanks @estherk15  - Would rather remove it from the Further Reading section then, this is more a piece of content to understand high level what the product does than deep dive